batcher/batcher-storage-ghostnet.tz

```diff
@@ -1,7 +1,8 @@
 (Pair (Pair (Pair (Pair "tz1ca4batAsNxMYab3mUK5H4QRjY8drV4ViL" {} {}) 600 10000)
             (Pair "tz1burnburnburnburnburnburnburjAYjjX" 0)
             10
-            {})
+            { Elt ""
+                  0x01101000011101000111010001110000011100110011101000101111001011110110100101110000011001100111001100101110011010010110111100101111011010010111000001100110011100110010111101010001011011010101011000110111010110100101001101001011011000110101100000110010010011010100111001110101011001010110100100111000011101000101101000110010011010000111001000110101010101010100100001001101010110100110011001110011011100000011100101000111011010110011011101010110011101010011010001011000011110000111011001101101011000100100011001110011010010100100111001000101 })
```

Comment:
What's this?

Reply:
That was my rubbish initial attempt at pre-populating the metadata bytes without reading the ligo docs regarding byte literals.  This is fixed if you look at the PR as a whole rather than commit by commit

Reply:
Right, but even that, what is it?

Reply:
You mean, what does it represent in the storage?  The ipfs link to the metadata json doc in bytes.